minifi-toolkit/minifi-toolkit-configuration/src/test/resources/InvokeHttpMiNiFiTemplateTest.yml

```diff
@@ -255,8 +273,8 @@ Remote Processing Groups:
   timeout: 30 sec
```

Comment:
The input ports already have ids, which is what should matter.  I think we can add some validation though.